test/suites/dev/moonbase/test-staking/test-staking-locks4.ts

```diff
@@ -3,8 +3,8 @@ import { beforeAll, describeSuite, expect } from "@moonwall/cli";
 import { GLMR, MIN_GLMR_STAKING, alith, generateKeyringPair } from "@moonwall/util";
 
 describeSuite({
-  id: "D023480",
-  title: "Staking - Locks - candidate balance is locked",
+  id: "D013478",
```

Comment:
This change was also caused by a conflict, we should revert it to keep the test id order.